nfs/docs/usage.md

```diff
@@ -7,6 +7,9 @@ Edit the `provisioner` field in `deploy/kubernetes/class.yaml` to be the provisi
 ### Parameters
 * `gid`: `"none"` or a [supplemental group](http://kubernetes.io/docs/user-guide/security-context/) like `"1001"`. NFS shares will be created with permissions such that pods running with the supplemental group can read & write to the share, but non-root pods without the supplemental group cannot. Pods running as root can read & write to shares regardless of the setting here, unless the `rootSquash` parameter is set true. If set to `"none"`, anybody root or non-root can write to the share. Default (if omitted) `"none"`.
 * `rootSquash`: `"true"` or `"false"`. Whether to squash root users by adding the NFS Ganesha root_id_squash or kernel root_squash option to each export. Default `"false"`.
+* `allSquash`: `"true"` or `"false"`. Whether to all usears should be squashed to anonUID and anonGID by adding the NFS Ganesha all_squash or kernel all_squash option to each export. Default `"false"`.
+* `anonUID`: `"none"` or `"integer"`. When rootSquash or allSquash is set, this uid will be used for  squashed users. If set to `"none"`, default of the server will be used.
```

Comment:
Will be worth mentioning for anonUID and anonGID, a similar to the gid description:

> Default (if omitted) `"none"`

Reply:
Was the last change what you requested?